aws/david_tutorial/trust/orion-ld/values.yaml

```diff
@@ -0,0 +1,100 @@
+orion:
+  deployment:
+    image: 
+      tag: 1.1.0
+  broker:
+    logging:
+      level: DEBUG
+    db:
+      auth: 
+        user: root
+        mech: "SCRAM-SHA-1"
+      hosts:
+        - mongodb-trust
+      user: root
+      existingSecret:
+        name: mongodb-secret
```

Comment:
This secret does not exist, see above

Reply:
I've pushed the 'sealed' secret. Is this the one that is needed?